lms/static/js/views/fields.js

```diff
@@ -129,7 +159,8 @@
                 this.lastSuccessMessageContext = context;
 
                 setTimeout(function () {
-                    if ((context === view.lastSuccessMessageContext) && (view.getNotificationMessage() === successMessage)) {
+                    if ((context === view.lastSuccessMessageContext) &&
+                        (view.getNotificationMessage() === successMessage.toString())) {
```

Comment:
I feel like the `===` and need for `toString()` here means a bug may have been introduced?  Why will getNotificationMessage() no longer be the successMessage() as is?  Does one now return an HTML Snippet, and the other not?  Shouldn't they stay the same?